performance-metrics/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
If it is nearcore-specific crate, let's name it with `near-` prefix and move to the `utils` folder, otherwise, I vote for extracting it to a standalone crate with its own git repo and publish on crates.io

Reply:
I agree, I renamed it to near-performance-metrics and moved it to utils.

Reply:
I would propose to keep it in utils for now, we can extract it to cargo.io later